### ISSUE TITLE
FW: Aging Differences From Aavaz

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#538]
In GitLab by @kdjstudios on Oct 23, 2017, 08:00

**Submitted by:** "Tim Traylor" <tim.traylor@answernet.com>

**Helpdesk:** http://www.servicedesk.answernet.com/profiles/ticket/4024999

**Server:** Internal

**Client/Site:** All

**Account:** All (Accounts on attached list)

**Issue:** 

Hi Sumeet,
 

We would like to get additional info on the AR issue, and would ask for a dump/report of the following:
1.       Compare the invoice balance to the AR as Invoiced report to ensure the balances match.
2.       Compare the AR to the ledger balances on the accounts.
 

Also, referring to the attached spreadsheet, I have questions I hope you can answer:
1.       D-061-2021: How could this account be dormant for years and suddenly have a value in the current aging bucket?
2.       T7201: Why does the account not show up on the AR as Invoiced?
3.       123-E13999: How did the system calculate $36 less in the ‘Current’ aging bucket than it should be?  (I’m not questioning the fact that the dollar amounts are in the wrong aging buckets J ).
4.       071-7518: How did the system calculate $65.10 more in the ‘Current’ than what it should be?
 

Items 3 and 4 make no sense to me because there are no charges or line items that seem to match these amounts. So it’s like it just randomly came up with a miscalculation.
 

Thanks!

Hi Gary,

Following up on the aging issues we started discussing in pipeline last week.  These are the ones Aavaz provided which show where the total due on the invoice doesn’t match the aging buckets on the invoice.  I attached the sheet if you’re curious.
 

Summarizing:
·         My research showed that the credit balance accounts are properly represented on the standard AR report, and their credit balances show in the current bucket.
·         The closed accounts with credits seem valid and to clear them they need debit adjustments.
·         Eliminating closed leaves 374 accounts.
·         The amount the invoice says is owed (ignoring the aging buckets) looks accurate.
·         The open accounts can be off in overstating or understating the AR buckets as compared to the actual due. In other words, there’s no pattern that is always resulting in AR being too high or AR being too low – it can go either way.
 

I met with Rich this AM and reviewed my findings and he suggested Aavaz run two comparisons for us:
1.       Compare the invoice balance to the AR as Invoiced report to ensure the balances match.
2.       Compare the AR to the ledger balances on the accounts.
 

Additionally, I need Aavaz to explain why we’re seeing differences between total AR on the invoice and the actual amount due.  This isn’t a case of something being in the wrong aging bucket, it’s a difference in the total dollars. We also have at least one open credit balance account that is not showing on the AR as invoiced, which they also need to research.
 

In short, we have nothing actionable yet and need additional answers from Aavaz before we decide if there’s anything (other than calculation fixes) that need to be done.
 

Attached File: [accounts_with_faulty_aging_calculation__1_.xls](/uploads/05378b6680e92400517e62404f0e746c/accounts_with_faulty_aging_calculation__1_.xls)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 26, 2017, 09:24

This needs to be discussed next week.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Oct 27, 2017, 23:21

@kace.coady @kdjstudios @tim.traylor @abakshi 

**Time Spent 1hr**

* We would like to get additional info on the AR issue, and would ask for a dump/report of the following:

 * Compare the invoice balance to the AR as Invoiced report to ensure the balances match.

 * Compare the AR to the ledger balances on the accounts.

  * Here is the dump [aging_differences.csv](/uploads/a98af6c0b10d86d57eaf60d63522731b/aging_differences.csv)


Also, referring to the attached spreadsheet, I have questions I hope you can answer:

1. D-061-2021: How could this account be dormant for years and suddenly have a value in the current aging bucket?  (**this is the first time I have encountered this calculation I'll get back to you on that but right now I have no Idea how system calculated this figure**)

![Screenshot-2](/uploads/569dec6920240807a9688f9abffe441e/Screenshot-2.png)

1. T7201: Why does the account not show up on the AR as Invoiced?  (**Instead of _ALL_ filter if we use _Balance Only_ filter on AR as Invoiced this account will not show up**)

1. 123-E13999: How did the system calculate $36 less in the ‘Current’ aging bucket than it should be?  (I’m not questioning the fact that the dollar amounts are in the wrong aging buckets J ).   

1. 071-7518: How did the system calculate $65.10 more in the ‘Current’ than what it should be?  

**for 3 & 4 the aging bucket calculation depend on previous invoice balance and that was not accurate as well. Right now it won't be possible to track previous invoice balance at the time of invoice generation**

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 30, 2017, 10:29

@smasih @tim.traylor @kace.coady 

In summary, I just want to confirm that the faulty aging is due to incorrect balances on previous invoices. If so, What are the solutions to correct this issue?

My thoughts, I believe from previous conversations are possibly the following:
1. Wait until the incorrect invoice balances are over 120 days then they will not be used in calculation and the aging will automatically correct itself. This however still have incorrect aging then for about 120 days. - Or because the aging is using incorrect values it will still generate bad values forever?
2. Manually edit the incorrect invoices with approval from the site and operations. This will change historical data and may present unexpected charges for the clients. - Any other concerns I may have missed?

Any other solutions that I am not thinking of?

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Oct 31, 2017, 24:41

@kdjstudios 

Totally agree with you Kyle and yes, there is no other solution that I can think of.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 31, 2017, 07:56

@smasih @kace.coady 

Sorry could you clarify, is the only way to fix aging by manually editing the accounts invoice balances? Or does the first thought fix aging as well?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Oct 31, 2017, 09:11

The second thought to manually edit the invoice balances would be recommended. Due to less time and insuring that the account is corrected.

The first thought, it was discussed that it is believed would automatically fix it self after the 120 days (or when the incorrect invoice balance is no longer being taken into calculation)

We will wait to speak with Cori, until Tim is back and then proceed with approval from operations to make any manual changes.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 10, 2017, 11:39

@tim.traylor @kace.coady 

I believe we are waiting to hear back from Cori or have this discussion with Cori to find a appropriate resolution to handles these accounts. Could we get an update please?

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Nov 12, 2017, 22:21

Moving to next milestone.

**Comment by kdjstudios** [comment]
In GitLab by @tim.traylor on Nov 13, 2017, 06:54

@kdjstudios Hi Kyle, we requested additional reporting from Aavaz before moving forward.  Rich and I reviewed this - I don't believe Cori was involved at all so far.  Thx, Tim

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 13, 2017, 09:31

@tim.traylor Thanks for the update, and clarification that you are working with Rich not Cori.  I will be removing the pause label as we do need the additional reporting.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Nov 14, 2017, 23:31

Hi [Tim] ( @tim.traylor ) & [Kyle] ( @kdjstudios ),   
Do we need to send any additional reporting? If yes, then can we have a prototype of the same.  
 
Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 15, 2017, 08:17

@tim.traylor 

Would you please provide @pchaudhary with the information regarding the reports that were requested. I do not have any details on those reports.

**Comment by kdjstudios** [comment]
In GitLab by @pchaudhary on Dec 26, 2017, 05:58

Hi [Kyle] ( @kdjstudios ),   
I think this is resolved along with the #604.Please let us know if there is anything else to do for close this.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Dec 26, 2017, 12:02

@pchaudhary 

I believe you are correct, marking for close.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Mar 7, 2018, 03:26

closed
